### ISSUE TITLE
A11Y: Buttons in AboutDialog need to have a bigger tap target

### ISSUE THREAD

**Issue by robertoscaramuzzi** [flutter/flutter#21002]
Our A11Y review pointed out that the touch target for the VIEW LICENCES and CLOSE buttons in AboutDialog are 35 dp, but they should be at least 48 dp for accessibility purposes.

(note that the buttons themselves don't need to be taller, but the touch target needs to)

**Comment by robertoscaramuzzi** [comment]
Never mind. I was filing a bug from an old review, and didn't realize it had been fixed in the meantime.

Closing this issue.
